### DISCUSSION TITLE
Device file for Calibo Smart Fan

### DISCUSSION THREAD

**Discussion by darro-gbn** [make-all/tuya-local#2377] [Show and tell]
Hi All, 

Firstly, Thanks the all the team members here.

I hope this is the right place for this.

I've edited an existing yaml file (thanks to that author) for the Calibo Smart fan distributed here in Australia (not sure elsewhere)

The Fan Speed, Direction, Mode and Light Brightness all seem to work.  The Light does have 3 colour temperatures, but this does not seemed to be exposed even via the Native Tuya app, so this does not work.

I hope it helps someone.

Cheers
Darro
[calibo_smart_ceiling_fan.zip](https://github.com/user-attachments/files/17252458/calibo_smart_ceiling_fan.zip)

**Comment by make-all** [comment]
The right place would be under Issues (New Device request), or as a pull request.

Discussions do not get tracked in the same way, so it is likely to get overlooked here.

**Comment by darro-gbn** [comment]
Thanks, It seems you've added my device file.  Thanks for that
I'll create a device request next time around.  

Cheers
Darren
